tensorflow/compiler/xla/client/lib/math.cc

```diff
@@ -341,7 +341,7 @@ XlaOp ErfInv(XlaOp x) {
       0.00943887047f,   1.00167406f,     2.83297682f};
 
   auto one = ScalarLike(x, 1.0);
-  auto w = -Log((one - x) * (one + x));
+  auto w = -Log1p(one - x * x);
```

Comment:
Log1p(x) = Log(1+x). I think the new line should look like this:
auto w = -Log1p(- x * x);

Can you put comment on the code to explain why Log1p is used instead of Log? In particular, why the input x here is small? What kind of test you have done?

Reply:
Thanks @bixia1, I've added comment and reference link about `Log1p`. According the [paper](https://people.maths.ox.ac.uk/gilesm/codes/erfinv/gems.pdf), there is no guarantee `x` is close to zero. But, replacing with `Log1p` won't change the result when `x` isn't small enough; and if it happens, the function is more precise.